### ISSUE TITLE
Crash in openSuSe when using Avoid Color Shift in Wavelet Tab

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#2856]
Originally reported on Google Code with ID 2876
```
When using RT, I get a crash (SEGFAULT) when using Wavelet Tab and I enable Gamut Controls-Avoid
Color Shift. This happens on any photo.

About this build:

Branch: default
Version: 4.2.279
Changeset: 14baa714cf93
Compiler: cc 4.8.3
Processor: undefined
System: Linux
Bit depth: 64 bits
Gtkmm: V2.24.4
Build type: release
Build flags:   -Werror=unused-label -fopenmp -Werror=unknown-pragmas -fmessage-length=0
-grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector -funwind-tables
-fasynchronous-unwind-tables -g
Link flags:    
OpenMP support: ON
MMAP support: ON

Debug output:

Program received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fffe2b74700 (LWP 16354)]
rtengine::ImProcFunctions::ip_wavelet(rtengine::LabImage*, rtengine::LabImage*, int,
rtengine::procparams::WaveletParams const&, rtengine::WavCurve const&, rtengine::WavOpacityCurveRG
const&, rtengine::WavOpacityCurveBY const&, rtengine::WavOpacityCurveW const&, rtengine::WavOpacityCurveWL
const&, LUT<float>&, bool, int) [clone ._omp_fn.4] ()
    at /usr/lib64/gcc/x86_64-suse-linux/4.8/include/xmmintrin.h:947
947       *(__v4sf *)__P = (__v4sf)__A;

GDB output:

(gdb) bt full
#0  0x00007ffff7aae902 in rtengine::ImProcFunctions::ip_wavelet(rtengine::LabImage*,
rtengine::LabImage*, int, rtengine::procparams::WaveletParams const&, rtengine::WavCurve
const&, rtengine::WavOpacityCurveRG const&, rtengine::WavOpacityCurveBY const&, rtengine::WavOpacityCurveW
const&, rtengine::WavOpacityCurveWL const&, LUT<float>&, bool, int) [clone ._omp_fn.4]
() at /usr/lib64/gcc/x86_64-suse-linux/4.8/include/xmmintrin.h:947
#1  0x00007ffff4080c36 in  () at /usr/lib64/libgomp.so.1
#2  0x00007ffff3c4c0a4 in start_thread () at /lib64/libpthread.so.0
#3  0x00007ffff398208d in clone () at /lib64/libc.so.6
(gdb) 

System: openSuSe 13.2 64-bit, AMD duo core, 2GB RAM and 2GB swap, no other applications
open.
```

Reported by `reginald.ricquier@scarlet.be` on 2015-08-10 15:03:33

**Comment by Beep6581** [comment]
Fix committed in rev 294c3cccdcbf07326b928708abee290b3bd74e67

**Comment by Beep6581** [comment]
Fix confirmed.
